fix: hpa enabled for testing

palla714 opened the pull request:
This PR is to merge the tested hpa config in the process of refactoring the badgr-server and simplifying the helm chart . The subbranch dev-charts-hpa is used for testing both hpa config and managed DB config 

![image](https://user-images.githubusercontent.com/53479552/165899652-53d86c9a-a35a-47cb-b08c-8821e944fab6.png)
![image](https://user-images.githubusercontent.com/53479552/165901361-83c36c64-6917-417a-81ba-5d0f066ac4cc.png)

Build Job URL
https://jenkins-labs-ci-cd.apps.dev.lxp.academy.who.int/job/badgr-server/job/dev-charts-hpa/5/

Build URL
http://dev-charts-hpa-badgr-server-badgr-labs-dev.apps.dev.lxp.academy.who.int/

krishnamadhavan commented:
@palla714 I had asked you to make changes to the Jenkinsfile, right ? I cannot see them in the changes.

palla714 commented:
> @palla714 I had asked you to make changes to the Jenkinsfile, right ? I cannot see them in the changes.

Hi.. I changed the Jenkinsfile in the dev-charts and then pulled them in my new branch.. After testing the run I noticed that I added the changes to dev-charts branch itself, but as it was working so did not discard it.
